src/module/actor/lancer-actor-sheet.ts

```diff
@@ -190,7 +190,7 @@ export class LancerActorSheet<T extends LancerActorType> extends ActorSheet<
       const flowArgs = flowElement.dataset.flowArgs;
       switch (flowType) {
         case "FullRepair":
-          break;
+          this.actor.beginFullRepairFlow(flowArgs);
```

Comment:
Needs to keep the `break;` on a following line, otherwise it will also execute the "Stabilize" case.

Reply:
good catch - updated in latest commit!